wowup-electron/src/assets/i18n/de.json

```diff
@@ -21,34 +21,34 @@
     },
     "WOWUP_UPDATE": {
       "DOWNLOADED_TOOLTIP": "WowUp Update installieren",
-      "INSTALL_MESSAGE": "Willst du WowUp neu Starten um das Update zu installieren?",
+      "INSTALL_MESSAGE": "Möchtest Du WowUp neustarten, um das Update zu installieren?",
       "INSTALL_TITLE": "WowUp Update bereit",
       "NOT_AVAILABLE": "Die aktuellste Version von WowUp ist bereits installiert",
-      "PORTABLE_DOWNLOAD_MESSAGE": "Möchtest du die letzte Portable-Version manuell herunterladen?\n\nDu musste die App manuell schließen und die neue Version über die Alte kopieren.",
+      "PORTABLE_DOWNLOAD_MESSAGE": "Möchtest Du die neueste Portable-Version manuell herunterladen?\n\nDu musst die App manuell schließen und die neue Version über die Alte kopieren.",
       "PORTABLE_DOWNLOAD_TITLE": "Manuelles Herunterladen erforderlich",
       "SNACKBAR_ACTION": "Update",
       "SNACKBAR_TEXT": "Eine neue Version von WowUp ist verfügbar",
       "TOOLTIP": "WowUp Update verfügbar",
-      "UPDATE_ERROR": "WowUp konnte nicht nach Updates suchen"
+      "UPDATE_ERROR": "WowUp konnte das Update nicht finden"
```

Comment:
not correct, WowUP does not know if there is an update, so "suchen" and not "finden"